crates/router/src/connector/paypal/transformers.rs

```diff
@@ -136,6 +151,22 @@ impl From<&PaypalRouterData<&types::PaymentsAuthorizeRouterData>> for ItemDetail
     }
 }
 
+impl From<&PaypalRouterData<&types::PaymentsPostSessionTokensRouterData>> for ItemDetails {
+    fn from(item: &PaypalRouterData<&types::PaymentsPostSessionTokensRouterData>) -> Self {
+        Self {
+            name: format!(
+                "Payment for invoice {}",
+                item.router_data.connector_request_reference_id
+            ),
+            quantity: 1,
```

Comment:
Any reason this is hardcoded?

Reply:
Initially we had only 1 order, so we are treating whole order as single line items in order